packages/e2e/src/specs/messages/messages-thread-creation.e2e.ts

```diff
@@ -0,0 +1,333 @@
+      await browser.keys("\ue003")
+    }
+    await MessagesConversationPage.insertTextToSearchContactInput(testNumber)
+
+    const recipientInput = await MessagesConversationPage.searchContactsInput
+    await expect(recipientInput).toHaveValue(testNumber)
+  })
+  it("Check thread name is 'New conversation'", async () => {
+    const thaedName = await MessagesPage.threadRow
+    await expect(thaedName).toHaveText("New Conversation")
+  })
+
+  it("Click on recipient input field and send Enter key to lock it and create a thread", async () => {
+    waitForClickableAndClick(await MessagesConversationPage.messageTextInput)
+    await browser.keys("\uE007")
+    const recipientInput = await MessagesConversationPage.searchContactsInput
+    await expect(recipientInput).not.toBeClickable()
+  })
+
+  it("Check thread name is updated with recipient number", async () => {
+    const conversationName =
+      await MessagesConversationPage.conversationRecipientNameText
+    const recipientText = await conversationName.getText()
+
+    const threadName = await MessagesPage.threadRow
+    await expect(threadName).toHaveText(recipientText)
+  })
+  it("Delete empty thread with selection manager", async () => {
+    await MessagesPage.hoverOverThreadRow()
+    await waitForClickableAndClick(await MessagesPage.threadCheckbox)
+    await waitForClickableAndClick(
+      await MessagesPage.selectionManagerDeleteButton
+    )
+    await ModalMessagesPage.clickConfirmDeleteButton()
+
+    const emptyThreadList = MessagesPage.threadScreenEmptyList
+    await emptyThreadList.waitForDisplayed({ timeout: 8000 })
+    await expect(emptyThreadList).toBeDisplayed()
+  })
+
+  after(async () => {
+    try {
+      await waitForClickableAndClick(
+        await MessagesConversationPage.threadDetailScreenCloseButton
+      )
+      await deleteConversationOnThreadList()
+      await waitForClickableAndClick(await NavigationTabs.contactsTab)
+      await waitForClickableAndClick(await NavigationTabs.messagesTab)
+    } catch (error) {
+      console.log(error)
+    }
+  })
+})
+
+describe("New thread creation scenarios - no contacts & thread exists", () => {
+  const existingThreadPhoneNumber = "98765"
+  const existingThreadMessageText = "Existing thread msg"
+  before(async () => {
+    // Waiting for device connected through USB
+    await browser.executeAsync((done) => {
+      setTimeout(done, 10000)
+    })
+    await ModalGeneralPage.clickUpdateAvailableModalCloseButton()
+    await waitForClickableAndClick(await NavigationTabs.messagesTab)
+    await sendMessage(existingThreadPhoneNumber, existingThreadMessageText)
+  })
+  it("input recipient number of existing thread and press enter ", async () => {
+    await waitForClickableAndClick(await MessagesPage.newMessageButton)
+    await MessagesConversationPage.insertTextToSearchContactInput(
+      existingThreadPhoneNumber
+    )
+    await browser.keys("\uE007")
```

Comment:
Maybe this '\uE007' could be saved in constant? :) And others browser keys too :)